doc/src/manual/performance-tips.md

```diff
@@ -1443,8 +1443,8 @@ Interpreting the output of [`@code_warntype`](@ref), like that of its cousins [`
 Your code is being presented in form that has been heavily digested on its way to generating
 compiled machine code. Most of the expressions are annotated by a type, indicated by the `::T`
 (where `T` might be [`Float64`](@ref), for example). The most important characteristic of [`@code_warntype`](@ref)
-is that non-concrete types are displayed in red; in the above example, such output is shown in
-uppercase.
+is that non-concrete types are displayed in red; since this document is written in Markdown, which has no color, 
```

Comment:
This is the trailing whitespace that's causing the red ❌.

```suggestion
is that non-concrete types are displayed in red; since this document is written in Markdown, which has no color,
```